src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -682,6 +687,17 @@ internal void ComputeFrameUpdate()
             {
                 directionalLight.Direction = v;
             }
+
+            // Occasional update of scene items, including
+            // distance-based transparency sorting.
```

Comment:
Ok, "dumb, but I don't think this is very descriptive." Are you sure you want me to call you that all the time? It's pretty long. I'll probably make a mistake every once and a while.